src/crawlee/_request.py

```diff
@@ -127,15 +127,17 @@ class BaseRequestData(BaseModel):
     method: HttpMethod = 'GET'
     """HTTP request method."""
 
-    headers: Annotated[HttpHeaders, Field(default_factory=HttpHeaders())] = HttpHeaders()
+    headers: Annotated[HttpHeaders, Field(default_factory=HttpHeaders)] = HttpHeaders()
     """HTTP request headers."""
 
     query_params: Annotated[HttpQueryParams, Field(alias='queryParams', default_factory=dict)] = {}
```

Comment:
We should probably get rid of this one, or make it a property that changes the `url` field. Unfortunately, Apify API does not accept `queryParams`.

Reply:
Yeah... Opened a new issue for it - #615.